homeassistant/components/zwave/__init__.py

```diff
@@ -600,14 +601,36 @@ def start_zwave(_service_or_event):
     return True
 
 
-class ZWaveDeviceEntity:
+class ZWaveDeviceEntity(Entity):
```

Comment:
We should not do this. ZWaveDeviceEntity is what is getting mixed in with the abstract base class that the platform provides.

For example the Z-Wave light entity will inherit ZWaveDeviceEntity and Light ABC:

```python
class ZwaveDimmer(zwave.ZWaveDeviceEntity, Light):
```

So inheritance from `Entity` should be removed.